### PR TITLE
feat(icon): add vyper icon

### PR DESCRIPTION
**Pull request by itsacoyote**
This adds the Vyper logo from https://vyperlang.org/

**Changes proposed:**

- [x] Add
- [ ] Delete
- [ ] Fix
- [ ] Prepare

### CODE REVIEW

File: src/iconsManifest/supportedExtensions.ts [hunk 1]
```diff
@@ -5386,6 +5386,12 @@ export const extensions: IFileCollection = {
       filename: true,
       format: FileFormat.svg,
     },
+    {
+      icon: 'vyper',
+      extensions: [],
```
Comment (trocher) [inline src/iconsManifest/supportedExtensions.ts:5391]:
```suggestion
      extensions: ['vyi'],
```

Comment (itsacoyote) [reply src/iconsManifest/supportedExtensions.ts:5391]: I don't believe this is a valid extension for vyper?  Is there an example of this usage?

Comment (itsacoyote) [reply src/iconsManifest/supportedExtensions.ts:5391]: Ah, I found it, looks like it is for interfaces: https://docs.vyperlang.org/en/latest/interfaces.html#standalone-interfaces

**Review by robertohuertasm — APPROVED** [review]
LGTM 🚀 Thanks for your contribution ❤️

**Pull request merged**